_projects/AIC018-Cerebriu-Brain_MRI_Workflows.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: project_page
+title: Smart brain MRI protocols 
+status: Governance
+image: /assets/img/blog/nhs_logo.png
+summary: Evaluation of brain MRI smart protocoling software
+csc-lead: <a href="/team_member/Laurence.html">Laurence</a>
+modality: MRI
+pathology: Neurology
+rationale: A consequence of the ever-increasing demand for diagnostic MR imaging is additional workload and stress placed on MRI departments. MRI deparements need to operate as efficiently as possible. A smart protocoling application for brain MRI data has the potential to improve the efficiency of the clinical MRI service and improve patient prioritisation. This application may also reduce scan time, re-examinations rates and non-acute admission to ward environments.
+patient-pathway: Patients who present to their GP with diffuse neurological symptoms (headaches, dizziness) are referred to GSTT for brain MRI. In current clinical practice, a radiologist will review these images and if an infarct is spotted, the radiologist must re-call the patient for additional MRI scans. Instead, with a smart protocol in place, the infarct can be flagged in real-time for the attention of the radiologist and additional follow-up scans can be completed in the same appointment slot.   
```

Comment:
@lucy-funnell – You have written a fair amount of text here, but it's not actually on the website because the `patient-pathway` frontmatter isn't called in the text below.

Did you want this text on the website?

Reply:
Hi @tomaroberts yes I have left the name Cerebrui out as this is the name of the company we are working with, but its the name i use to refer to the project.

All the project pages have Rationale and Patient pathway sections. I have populated them but i don't think we had decided what format (how this would look visually) to use on the website. 
I happy to have it as a free text section under the heading Clinical Lead but before Project Plan, would that be appropriate?

Reply:
Understood, thanks. Pop in the body of the text as well and then we can adapt the formatting to be more consistent going forward.